crates/store/re_remote_store_types/proto/rerun/v0/common.proto

```diff
@@ -161,3 +161,15 @@ enum SparseFillStrategy {
     NONE = 0;
     LATEST_AT_GLOBAL = 1;
 }
+
+// Error codes for application level errors
+enum Code {
+    // unused
+    _NO_ERROR = 0;
```

Comment:
If it is in fact unused lets call it `_UNUSED`.  "NO_ERROR" still sounds like a way of indicating a success-type.